### ISSUE TITLE
Find symbol hotkey catching when RD is not active

### ISSUE THREAD

**Issue by IvenBach** [rubberduck-vba/Rubberduck#5320]
Version 2.4.1.20935
OS: Microsoft Windows NT 10.0.18362.0, x64
Host Product: Microsoft Office 2013 x86
Host Version: 15.0.5023.1000
Host Executable: EXCEL.EXE

**Description**
This is an intermittent issue. When using the hotkey `Ctrl+T` to open a new browser tab, which is also the same as Find Symbol, RD is latching onto it. RD displays the Find Symbol dialog even though the VBA IDE isn't the active window.

**To Reproduce**
1. Have VBA IDE open in background
2. Active window is a browser
3. `Ctrl+T` to open a new tab. Occassionally no tab is created since RD is grabbing the hotkey for Find Symbol and displays that dialog.

**Expected behavior**
When RD is in the background none of its hotkeys should be listening.

**Additional context**
IIRC this has come up before and at that time had been resolved. As this is not 100% reproducible I'll update ~~if~~ when I can determine a consistent cause.

**Comment by Greedquest** [comment]
Yes I frequently switch to Excel to find the "find symbol" dialogue open; didn't realise this was the cause, but sounds plausible

**Comment by interservices** [comment]
Ah thanks IvenBach!
I came to this page after finding for the nth time today an unrequested RD "Find Symbol" open after the alt+tab to VBE.  And seconds before I had to press Ctrl+T in Edge to make it work!

So 3 years after your messages the problem is still here, with the latest version of RD and Excel 64bit.

**Comment by retailcoder** [comment]
Won't fix the hotkey hooking, but note that all hotkeys can be configured in the settings dialog; if Ctrl+T is disruptive, consider changing it to another, less-used key, or tick the "Shift" box to make it Ctrl+Shift+T.

**Comment by interservices** [comment]
Ah, thanks for the suggestions!
I eventually changed the RD shortcut of "Find symbol" to Ctrl+Alt+T as I also sometime use the Edge/Chrome Ctrl+Shift+T for restoring closed tabs and there's another feature of RD that defaults to Ctrl+Shift+T (Test Explorer).

BTW: in my previous message I wrongly wrote "press Ctrl+T in Edge to make it work" insted than "press Ctrl+T two times in Edge to make it work".  Edited now.
